WcaOnRails/app/webpacker/components/CompetitionOverview/CompetitionTable.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import ReactMarkdown from 'react-markdown';
+
+import I18n from '../../lib/i18n';
+
+function calculateDayDifference(comp, mode) {
```

Comment:
Would prefer this move to `utils/`

Reply:
so generalizing this function and moving it into one of the existing `utils/` js files?